Resources/translations/SonataNewsBundle.fr.xliff

```diff
@@ -427,6 +427,14 @@ Liens de modération rapide :
                 <source>group_classification</source>
                 <target>Classification</target>
             </trans-unit>
+            <trans-unit id="sonata_news_block_recent_comments">
```

Comment:
btw. why is the id different to the source key?

i think in all other files we keep them the same, don't we?